com.unity.robotics.urdf-importer/Runtime/Extensions/MaterialExtensions.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.Robotics
+{
+    public static class MaterialExtensions
+    {
+        private static string[] standardShaders = { "Standard" };//, "UI/Default" };
+        private static string[] hdrpShaders = { "HDRP/Lit" };//, "UI/Default" };
```

Comment:
shall we remove the unused code `UI/Default`?

Reply:
I kept it so that it hints us that this is an array and this can be done in the future if we are fixing a bug related to materials/shaders.

Reply:
URP support added in b9aabfc